### ISSUE TITLE
Empty result

### ISSUE THREAD

**Issue by VitaliyDemchuk** [IzakMarais/reporter#35]
When I generate pdf I get empty results. Only the date is displayed:
```
Thu Mar 1 23:22:00 UTC 2018
to
Fri Mar 2 11:22:00 UTC 2018
```
In Grafana reporter log i see:
```
2018/03/02 11:34:58 Report generated correctly
```

**Comment by IzakMarais** [comment]
Hmm that sounds like a bug (if the query is wrong, at least there should be a descriptive error somwhere). 

Can you provide some more detail? If you can attach the dashboard json file as well as the exact reporter query you ran, that would be ideal.

**Comment by VitaliyDemchuk** [comment]
@IzakMarais unfortunately I can not attach the dashboard. It consists of text panels and bars.
query report: `/api/report/{grafana_dashboard_name}?apitoken={grafana_api_token}&from=now-12h&to=now`

**Comment by VitaliyDemchuk** [comment]
@IzakMarais [i use docker.](https://github.com/IzakMarais/reporter/blob/master/util/docker-compose.yml#L19)
Maybe you should use another tag to build the image?

**Comment by IzakMarais** [comment]
I'll have to test this with the docker compose. Have you tried without Docker?

**Comment by IzakMarais** [comment]
What grafana version are you using? I also get this behaviour with Grafana 5 currently. After implementing https://github.com/IzakMarais/reporter/issues/28, this should work.

**Comment by VitaliyDemchuk** [comment]
@IzakMarais i dont tried used without docker. My Grafana Version v5.0.0-beta4 (commit: 1de8891)

**Comment by IzakMarais** [comment]
> My Grafana Version v5.0.0-beta4 (commit: 1de8891)

OK, that explains it then. If you try with Grafana 4.6 it should work. Otherwise wait for #28 to be implemented; I'm busy working on it now.

**Comment by VitaliyDemchuk** [comment]
@IzakMarais OK. Thx.

**Comment by IzakMarais** [comment]
This is fixed in v2.0.0
